config/crd/bases/http.keda.sh_httpscaledobjects.yaml

```diff
@@ -155,12 +155,6 @@ spec:
                     type:
                       description: Type of condition
                       enum:
-                      - Created
-                      - Terminated
-                      - Error
-                      - Pending
-                      - Terminating
-                      - Unknown
```

Comment:
Currently, we have too many conditions without a huge sense, I've just removed all except `Ready` and I plan to give some love in the future to rethink this to make it useful for users

Reply:
https://github.com/kedacore/http-add-on/issues/902